website/pages/_document.tsx

```diff
@@ -0,0 +1,35 @@
+import Document, { Html, Head, Main, NextScript } from 'next/document';
+import * as React from 'react';
+import { renderStatic } from 'utils/renderer';
+
+export default class AppDocument extends Document {
```

Comment:
can you add docstring with a link to the Next.js emotion example here

Reply:
yep